package.json

```
@@ -91,17 +91,17 @@
     "@bbc/psammead-inline-link": "^0.3.4",
     "@bbc/psammead-media-indicator": "^0.2.1",
     "@bbc/psammead-paragraph": "^1.0.2",
-    "@bbc/psammead-section-label": "^0.1.0-alpha.0",
+    "@bbc/psammead-section-label": "^0.1.1",
     "@bbc/psammead-sitewide-links": "^0.3.9",
-    "@bbc/psammead-story-promo": "^0.2.0-alpha.6",
+    "@bbc/psammead-story-promo": "^0.2.0-alpha.8",
     "@bbc/psammead-story-promo-list": "^0.1.0-alpha.2",
     "@bbc/psammead-styles": "^0.4.1",
-    "@bbc/psammead-timestamp-container": "^1.1.0",
+    "@bbc/psammead-timestamp-container": "^1.2.0",
```

```suggestion
    "@bbc/psammead-timestamp-container": "^1.2.1",
```
Version 1.2.0 had a bug with the locales that was fixed with 1.2.1. I can fix in a follow-up PR, though, since it only affects services that are not News (and consequently not yet live). Just stating it here for visibility.

---

@sareh please do in a followup PR, or catch the dependabot PR